### PR TITLE
Add tests for docker commands

### PR DESCRIPTION
**Pull request by Nyarum**
Work for #86 

Run tests via Makefile (For example):
- `make test_docker`

Output, like:
```
go test -v ./cmd/docker"
=== RUN   TestDockerLogin
----->  Configuring client
        Project 'Test' with id='57be1db75b0f340007436c8b'
--- PASS: TestDockerLogin (2.14s)
PASS
ok      github.com/iron-io/ironcli/cmd/docker   2.189s
```

### CODE REVIEW

File: cmd/docker/docker_login_test.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	)
+
+	common.SetSettings(settings)
+	dockerLogin := NewDockerLogin(settings)
+	dockerLogin.Email = email
+	dockerLogin.Username = username
+	dockerLogin.Password = password
+
+	err := dockerLogin.login()
+	assert.Nil(err)
```
Comment (nikhilm) [inline cmd/docker/docker_login_test.go]: Please don't pull in a dependency for a 4 line function.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+build:
+	go build main.go
+
+.PHONY: test_docker
+test_docker:
+	go test -v ./cmd/docker --email="${email}" --password="${password}" --username="${username}"
```
Comment (nikhilm) [inline Makefile]: Also put this in circle.

**Comment by Nyarum** [comment]
@nikhilm please don't check this. It's not ready, I set "in progress" label.

**Comment by nikhilm** [comment]
Could we please get per-command unit test PRs and stop this one before it gets too hard to review? Thanks.

**Comment by Nyarum** [comment]
@nikhilm then will be too many PRs.

**Comment by nikhilm** [comment]
Too many small PRs is better than any giant PR

On Thu, Sep 22, 2016 at 10:57 PM, Grigorij Eleskin <notifications@github.com

**Pull request closed without merging**